src/client/tile.cpp

```diff
@@ -82,11 +83,9 @@ std::string getImagePath(std::string path)
 {
 	// A NULL-ended list of possible image extensions
 	const char *extensions[] = {
-		"png", "jpg", "bmp", "tga",
-		"pcx", "ppm", "psd", "wal", "rgb",
-		NULL
+		"png", "jpg", "bmp", NULL
```

Comment:
And this is a little opportunistic edit zapping unsupported formats that we're no longer capable of loading.